pages/blog/posts/GSOD-docs-project-2024.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Update from the Google Season of Docs 2024 Project'
+date: '2024-09-03'
+tags:
+  - Google Season of Docs
+  - Documentation
+  - Community
+type: Documentation
+cover: /img/posts/2024/GSOD-cover/JSON-Schema-cover.png
```

Comment:
While the cover image is ok, I suggest you check what we did for other posts and search for a license-free image on Unsplash and use it.